core/org.osate.xtext.aadl2.properties/src/org/osate/xtext/aadl2/properties/util/GetProperties.java

```diff
@@ -944,22 +930,6 @@ public static double getProcessorMIPS(final ComponentInstance processor) {
 		return getMIPSCapacityInMIPS(processor, 0);
 	}
 
-	/**
-	 * compute MIPS for thread based on instructions per dispatch;
-	 *
-	 * @param threadinstance
-	 *            thread instance
-	 * @return MIPS or 0.0
-	 */
-	public static double getThreadExecutionIPDinMIPS(ComponentInstance threadinstance) {
```

Comment:
Public methods should not be removed without deprecating them fist. Also, add javadoc
`@deprecated will be removed in 2.6.0` (we search for such comments to find methods that can be removed). A comment why the deprecation and what to use instead shoud be there, too.

Same for other removed public methods